gpu/gpu_linux.go

```diff
@@ -48,8 +48,24 @@ var (
 	NvcudaMgmtName = "libcuda.so*"
 	NvmlMgmtName   = "" // not currently wired on linux
 	OneapiMgmtName = "libze_intel_gpu.so*"
+	VulkanMgmtName = "libvulkan.so*"
+	libcapMgmtName = "libcap.so*"
 )
 
+var VulkanGlobs = []string{
+	"/usr/lib/x86_64-linux-gnu/libvulkan.so*",
```

Comment:
This doesn't account for other arch's like aarch64. On my Raspberry Pi (see https://github.com/geerlingguy/ollama-benchmark/issues/1#issuecomment-2489622573), I get the error:

```
+ cp '/usr/lib//libvulkan.so*' ../build/linux/arm64/vulkan/bin/
cp: cannot stat '/usr/lib//libvulkan.so*': No such file or directory
llm/generate/generate_linux.go:3: running "bash": exit status 1
```

Reply:
Also unlikely to work on NixOS for example, as /usr/lib doesn't exist there (could use an environment variable?)